bundles/org.openhab.binding.tsmart/README.md

```diff
@@ -0,0 +1,53 @@
+# Tesla T-Smart Binding
+
+This binding is designed to connect to the Tesla T-Smart Smart Immersion Heater / Thermostat device https://www.tsmart.co.uk/.
+
+The device allows monitoring and control over the temperature and heating of water in a water cylinder.
+
+The build is derived from the API docs shared here https://community.home-assistant.io/t/telsa-t-smart-immersion-heater/345853 and some inspiration from https://github.com/britkat1980/tsmart.
+
+## Supported Things
+
+This supports a single thing type, the Tesla T-Smart Device.
+
+## Discovery
+
+The binding will discover Tesla T-Smart devices on your local network using a UDP Broadcast packet.
+
+## Thing Configuration
+
+| Name            | Type    | Description                           | Default | Required | Advanced |
+|-----------------|---------|---------------------------------------|---------|----------|----------|
+| hostname        | text    | Hostname or IP address of the device  | N/A     | yes      | no       |
+| refreshInterval | integer | Interval the device is polled in sec. | 60      | no       | yes      |
+
+## Channels
+
+| Channel         | Type               | Read/Write | Description                                |
+|-----------------|--------------------|------------|--------------------------------------------|
+| `power`           | Switch             | RW         | Power for the thermostat                   |
+| `mode`            | String             | RW         | Operating mode of the thermostat           |
```

Comment:
Either the docs is wrong or the thing-types.xml as the docs say it is RW and the xml has readonly=true